### PR TITLE
Remove `astropy` version check

### PR DESCRIPTION
**Pull request by ViciousEagle03**
The minimum required version of astropy is 5.2. 

https://github.com/astropy/asdf-astropy/blob/9348f899f57114b574b050ea59d3c74b28e498f6/pyproject.toml#L24

This PR removes the `if` check for the following:

https://github.com/astropy/asdf-astropy/blob/9348f899f57114b574b050ea59d3c74b28e498f6/asdf_astropy/converters/transform/tests/test_transform.py#L81

Originally discussed: #235

### CODE REVIEW

**Comment by ViciousEagle03** [comment]
Would you like me to add a changelog entry?

**Review by braingram — APPROVED** [review]
LGTM! Once the CI goes green I'll merge this and hopefully that should allow the CI to run on your other PR.

I secretly did this so you'd clean up this code :) (just kidding)

**Pull request merged**
